aiida/restapi/resources.py

```diff
@@ -204,6 +204,122 @@ def get(self, id=None, page=None):  # pylint: disable=redefined-builtin,invalid-
         return self.utils.build_response(status=200, headers=headers, data=data)
 
 
+class QueryBuilder(BaseResource):
+    """
+    "Abstract" QueryBuilder REST API resource.
```

Comment:
You are right that this is a bit different in that there is no correspondence of this resource with an AiiDA database entity.

Not sure "abstract" is the right word, though ...

Reply:
"Abstracted"? "Mock"? "Representation of a"?